app/models/miq_server/worker_management/monitor/validation.rb

```diff
@@ -4,22 +4,34 @@ module MiqServer::WorkerManagement::Monitor::Validation
   def validate_worker(w)
     return true if MiqEnvironment::Command.is_podified?
 
-    time_threshold   = get_time_threshold(w)
-    memory_threshold = get_memory_threshold(w)
+    if !worker_heartbeat_valid?(w)
```

Comment:
I'm not sure if there are strong feelings about the rubocop on these methods, but for some reason this reads way better to me than `unless worker_heartbeat_valid?(w)`